### DISCUSSION TITLE
The hacker likely exploited the cloud server's SSH key stored in my Actions Secrets and launched an attack on my server, resulting in data theft and a ransom demand.

### DISCUSSION THREAD

**Discussion by gaozewen** [community/community#81218] [Code Security]
### Select Topic Area

Bug

### Body

I placed my SSH private key in Actions secrets on GitHub to facilitate automatic deployment of my project. However, on the evening of December 18th, when the project was automatically deployed through GitHub Actions, the server crashed due to a memory overflow, and the cloud server indicated a remote login from an unfamiliar location. Based on this, I suspect that the hackers may have tampered with the server during this process. After restarting the server and logging in, I discovered that all the data in the MySQL database had been stolen by hackers, who then demanded BTC as ransom.

**Comment by Zoey115** [comment]
HAVE YOU LOST YOUR BITCOIN? DO YOU WISH TO RESTORE YOUR STOLEN BTC? GET IN TOUCH WITH CYBER ASSET RECOVERY

Approximately a few Months ago, I invested bitcoins worth $87,000 with the wrong platform, I believed that I had irreversibly lost my digital assets. However, just as I started to worry about the potential loss, a colleague of mine recommended Cyber Asset Recovery a highly reputable hacking group that specializes in precisely such scenarios. The remarkable service provided by Cyber Asset Recovery has far exceeded my expectations. Their support and dependability have been exceptional. They have tirelessly worked to locate and recover my digital currencies, leaving no stone unturned. I sincerely appreciate their dedication and the peace of mind they have brought me during this challenging time. If you or anyone you know has faced a similar situation of losing cryptocurrency sent to a fictitious wallet or being scammed by fake online investors, I wholeheartedly recommend contacting Cyber Asset Recovery. They can be reached via email: ( [Cyberassetrecovery@protonmail.com](mailto:Cyberassetrecovery@protonmail.com) ) Their team of experts is committed to providing efficient and reliable assistance in cryptocurrency recovery. Once again, I express my sincere gratitude to Cyber Asset Recovery for their outstanding service and for going above and beyond to support their clients. I am genuinely grateful for their expertise and professionalism. Website:   https://cyberassetrecovery.pro/ 
Telegram: +1 (501) 725–1653

**Comment by Zoey115** [comment]
HAVE YOU LOST YOUR BITCOIN? DO YOU WISH TO RESTORE YOUR STOLEN BTC? GET IN TOUCH WITH CYBER ASSET RECOVERY

Approximately a few Months ago, I invested bitcoins worth $87,000 with the wrong platform, I believed that I had irreversibly lost my digital assets. However, just as I started to worry about the potential loss, a colleague of mine recommended Cyber Asset Recovery a highly reputable hacking group that specializes in precisely such scenarios. The remarkable service provided by Cyber Asset Recovery has far exceeded my expectations. Their support and dependability have been exceptional. They have tirelessly worked to locate and recover my digital currencies, leaving no stone unturned. I sincerely appreciate their dedication and the peace of mind they have brought me during this challenging time. If you or anyone you know has faced a similar situation of losing cryptocurrency sent to a fictitious wallet or being scammed by fake online investors, I wholeheartedly recommend contacting Cyber Asset Recovery. They can be reached via email: ( [Cyberassetrecovery@protonmail.com](mailto:Cyberassetrecovery@protonmail.com) ) Their team of experts is committed to providing efficient and reliable assistance in cryptocurrency recovery. Once again, I express my sincere gratitude to Cyber Asset Recovery for their outstanding service and for going above and beyond to support their clients. I am genuinely grateful for their expertise and professionalism. Website: https://cyberassetrecovery.pro/
Telegram: +1 (501) 725–1653
